src/zenml/integrations/seldon/services/seldon_deployment.py

```diff
@@ -84,8 +84,10 @@ def get_seldon_deployment_labels(self) -> Dict[str, str]:
         labels = {}
         if self.pipeline_name:
             labels["zenml.pipeline_name"] = self.pipeline_name
+        if self.run_name:
+            labels["zenml.run_name"] = self.run_name
         if self.pipeline_run_id:
-            labels["zenml.pipeline_run_id"] = self.pipeline_run_id
+            labels["zenml.pipeline_run_id"] = self.run_name
```

Comment:
I don't really understand the logic here, why do we still need to add it to the labels for "pipeline_run_id"?